pkg/core/database/lite/transactions.go

```diff
@@ -202,8 +202,12 @@ func (t transaction) FetchDecoys(numDecoys int) []ristretto.Point {
 func (t transaction) FetchOutputExists(destkey []byte) (bool, error) {
 	return false, nil
 }
-func (t *transaction) StoreCandidateBlock(b *block.Block) error {
 
+func (t transaction) FetchOutputUnlockHeight(destkey []byte) (uint64, error) {
```

Comment:
Would it be possible to keep lite driver up-to-date so that it can be used for testing and benchmark purposes? Not urgent , can be done by another PR.

Reply:
Also a good point. I will open an issue.